ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/hosts.yml

```diff
@@ -3,6 +3,12 @@ all:
     CLEAN_UNIT_TESTS:
       hosts:
         cvp-instance-ips-cvaas
+    BACKBONE_UNIT_TESTS:
+      hosts:
+        backbone-node-1:
```

Comment:
please name the node after the feature they test. Do you need two of each type? You could use one of each type and then see how the same backbone interface data renders between them.

Reply:
Renamed the nodes after tested feature and removed 2 so we have only 1 of each type.